scripts/core/editor3/reducers/tests/suggestions_insert.spec.jsx

```diff
@@ -0,0 +1,197 @@
+import {EditorState, ContentState} from 'draft-js';
+import * as Highlights from '../../helpers/highlights';
+import reducer from '../suggestions';
+
+describe('editor3.reducers.suggestion.CREATE_ADD_SUGGESTION', () => {
+    it('CREATE_ADD_SUGGESTION - insert text on empty content', () => {
+        let i;
```

Comment:
Since  you are not using it anywhere else, put the initialization inside the parens of the loop